pkg/quarterdeck/config/config.go

```diff
@@ -50,6 +51,16 @@ type TokenConfig struct {
 	RefreshOverlap  time.Duration     `split_words:"true" default:"-15m"`     // $QUARTERDECK_TOKEN_REFRESH_OVERLAP
 }
 
+// Used by the rate limiter
+// Limit: represents the number of tokens that can be added to the token bucket per second
+// Burst: maximum number of tokens/requests in a "token bucket" and is initially full
+// Ttl: //number of minutes before an IP is removed from the ratelimiter map
+type RatelimitConfig struct {
+	Limit float64       `default0:"10"`
+	Burst int           `default0:"120"`
+	Ttl   time.Duration `default0:"5"`
```

Comment:
This might make configuration a little easier:

```suggestion
	PerSecond float64       `default:"20" split_words:"true"`
	Burst int           `default:"120"`
	TTL   time.Duration `default:"5"`
```

Would you also mind updating the tests with the new config variables?